jenkins/docker/windows/python.Dockerfile

```diff
@@ -0,0 +1,26 @@
+# escape=`
+
+ARG FROM_IMAGE=symbolplatform/symbol-server-compiler:windows-msvc-17
+
+FROM ${FROM_IMAGE}
+LABEL maintainer="Catapult Development Team"
+
+SHELL ["pwsh","-command", "$ErrorActionPreference = 'Stop';"]
+
+# Microsoft Visual C++ Redistributable packages for Visual Studio 2013 required by pyzbar
+ADD https://aka.ms/highdpimfc2013x64enu c:\temp\vc_redist.x64.exe
+RUN Start-Process -filepath C:\temp\vc_redist.x64.exe -ArgumentList "/install", "/passive", "/norestart" -Passthru | Wait-Process
+
+RUN Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser; `
+	(new-object net.webclient).DownloadFile('https://get.scoop.sh','c:\scoop.ps1'); `
+	$command='c:\scoop.ps1 -RunAsAdmin'; `
+	iex $command; `
+	del c:\scoop.ps1; `
+	scoop install python git shellcheck openssl cmake cygwin; `
+	python3 -m pip install --upgrade pip; `
+	python3 -m pip install --upgrade gitlint wheel
+
+# Install codecov
+RUN Invoke-WebRequest -Uri https://uploader.codecov.io/latest/windows/codecov.exe -Outfile c:\Windows\System32\codecov.exe
+
```

Comment:
i guess you don't need to restore default shell because no more commands are run here?

Reply:
Yeah, in the Javascript Dockerfile to set the PATH with ``setx`` the shell needs to be cmd.
The entry point is the same for both images